### PR TITLE
Adding reducers and actions - Bookstore Project

### PR DESCRIPTION
**Pull request by ouasamine**
## Overview 
In this milestone, I created all the reducers and actions that are going to be used by other components.
## Workflow
- Created a `redux/` folder to contain the actions, reducers and store configuration files.
- Created `books/books.js` and `categories/categories.js` inside `redux/`.
- Created `configureStore.js` inside `redux/`.
- Created actions creators & and reducers.
- Imported all the necessary tools in `configureStore.js` and configured the store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (TedLivist) [inline src/redux/books/books.js:2]: - [ ] The values of your action types should be according to ducks pattern. Kindly do that.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  ```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (TedLivist) [inline src/redux/categories/categories.js:1]: - [ ] The values of your action types should be according to the ducks' pattern. Kindly do that.

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ouasamine,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ouasamine** [comment]
@TedLivist  Thanks 👍🏽

**Review by cwaku — APPROVED** [review]
Hi @ouasamine 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ouasamine** [comment]
Thank you @cwaku.

**Pull request merged**